compiler/src/main/java/com/squareup/anvil/compiler/codegen/dagger/MapKeyCreatorGenerator.kt

```diff
@@ -178,6 +178,7 @@ internal class MapKeyCreatorGenerator : PrivateCodeGenerator() {
         className,
         properties.entries.map { it.value.callExpression }.joinToCode()
       )
+      .returns(className)
```

Comment:
Nice, coincidentally this is also needed for upgrading to newer KotlinPoet versions